[FastPR][Fluid][Hotfix] Couette test execution

rubenzorrilla opened the pull request:
**Description**
The Couette tests were not properly executed in the suites. This PR amends this.

rubenzorrilla committed:
Hotfix Couette test execution

sunethwarna commented on applications/FluidDynamicsApplication/tests/couette_flow_test.py:
```diff
@@ -51,6 +51,12 @@ def tearDown(self):
+        self._CustomizeSimulationSettings()
+        self.runTest()
+        self.tearDown()
+
```
Hi, just curious... Is there a specific reason why you use a seperate function to call it? Wouldn't it be possible by using default cals of unit tests to call `setUp`, `tearDown` methods?

rubenzorrilla replied:
The point is that these were not called... This is why I created the auxiliary function.

rubenzorrilla replied:
(it is possible that I did something wrong)

philbucher replied:
I would say that now `setUp` and `tearDown` is being called twice
IMO is worth to check the reason, this could lead to more problems than it solved

rubenzorrilla replied:
This what I thought but seems that they're not being called. I'll go through it.

sunethwarna commented on applications/FluidDynamicsApplication/tests/couette_flow_test.py:
```diff
@@ -134,8 +140,5 @@ def _AddReferenceValuesCheck(self):
 if __name__ == '__main__':
     test = CouetteFlowTest()
-    test.setUp()
     # test.testCouetteFlow2DSymbolicStokes()
```
Do we have to deactivate the test in here? Can't we have them moved to nighly if required in the `test_FluidDynamicsApplication.py` file?

rubenzorrilla replied:
mmm... I think I'm not getting your point... This is to run this test only without the need of running the suite (again I might be wrong).

rubenzorrilla committed:
Merge branch 'master' into fluid/hotfix-couette-tests-execution

sunethwarna commented on applications/FluidDynamicsApplication/tests/couette_flow_test.py:
```diff
@@ -134,8 +136,5 @@ def _AddReferenceValuesCheck(self):
 
 if __name__ == '__main__':
     test = CouetteFlowTest()
```
```suggestion
    UnitTest.main()
```

rubenzorrilla committed:
Properly fixing test

rubenzorrilla committed:
Update applications/FluidDynamicsApplication/tests/couette_flow_test.py

Co-authored-by: Suneth Warnakulasuriya <suneth.warna@gmail.com>

sunethwarna committed:
minor

Shouldn't it be like this? (I had a typo in the suggestion, so i made the correction :) )

It will run both tests if this script is executed directly.

rubenzorrilla commented on applications/FluidDynamicsApplication/tests/couette_flow_test.py:
```diff
@@ -135,6 +135,4 @@ def _AddReferenceValuesCheck(self):
-    UnitTest.main()
-    # test.testCouetteFlow2DSymbolicStokes()
-    test.testCouetteFlow2DSymbolicNavierStokes()
+    KratosUnittest.main()
```
As these are small I'm fine with running both.

philbucher reviewed (approved):
looks good to me from the testing perspective
